src/update_disk.c

```diff
@@ -50,7 +63,13 @@
 #define MAX_FAILURES 4
 
 #define IMAGE_PRELOAD_ADDRESS 0x5000100
-
+/**
+ * @brief RAM function for starting the boot process.
```

Comment:
It is irrelevant here if the function can be executed from RAM when `RAM_CODE=1`. Please remove "RAM" from "RAM function [...]"